new_course/1_programming_fundementals/methods.md

```diff
@@ -259,21 +292,21 @@ puts whisper #=> "hello everybody"
 Writing `whisper.downcase!` is the equivalent of writing `whisper = whisper.downcase`.
 
 ### Assignment
-
 <div class="lesson-content__panel" markdown="1">
+
 1. For a good introduction to all the different concepts related to methods, read the [Methods chapter](https://launchschool.com/books/ruby/read/methods) from Launch School's *Introduction to Programming with Ruby*. Make sure to do the exercises at the end of the chapter too!
 2. To get a different take, read the part of the Objects section that discusses [Methods](http://ruby-for-beginners.rubymonstas.org/objects/methods.html) from Ruby Monsta's *Ruby for Beginners*.
-3. For more depth on how you can write your own classes, read the section on [Writing Methods](http://ruby-for-beginners.rubymonstas.org/writing_methods.html) from Ruby Monsta's *Ruby for Beginners*. 
+3. For more depth on how you can write your own methods, read the section on [Writing Methods](http://ruby-for-beginners.rubymonstas.org/writing_methods.html) from Ruby Monsta's *Ruby for Beginners*. 
 </div> 
 
 ### Additional Resources
 This section contains helpful links to other content. It isn't required, so consider it supplemental for if you need to dive deeper into something.
 
   - For a deeper look at methods, read the [Methods chapter](http://ruby.bastardsbook.com/chapters/methods/) from the Bastards Book of Ruby. Try to complete the exercises throughout the chapter.
-  - For more discussion n the subtle differences between methods and functions and how they can differ by language, here is a [handy explanation on Stack Overflow](https://stackoverflow.com/questions/155609/difference-between-a-method-and-a-function).
+  - For more discussion on the subtle differences between methods and functions and how they can differ between programming languages, here is a [handy explanation on Stack Overflow](https://stackoverflow.com/questions/155609/difference-between-a-method-and-a-function).
 
 ### Knowledge Check
-This section contains questions for you to check your understanding of this lesson. If you're having trouble answering the questions below on your own, review the material above to find the answer.
+This section contains questions you can use to check your understanding of this lesson. If you're having trouble answering the questions below on your own, review the material above to find the answer.
```

Comment:
The language in the original makes reviewing these questions sound like more of a requirement and less of a suggestion, which I feel is preferable. It's also a change that we would have to make across many lessons, so I'm not sure it's worth it. >.<